Block/Info/Boleto.php

```diff
@@ -29,4 +29,14 @@ class Boleto extends AbstractInfo
      * @var string
      */
     protected $_template = 'Adyen_Payment::info/adyen_boleto.phtml';
+
+    /**
+     * @param $data string
+     * @return string
+     */
+    public function getPaymentActionData($data)
+    {
+        $paymentAction = $this->getMethod()->getInfoInstance()->getAdditionalInformation('action');
+        return !empty($paymentAction[$data]) ? $paymentAction[$data] : '';
```

Comment:
Usually it's easier to understand the not short syntax. Both are good but maybe it's easier to read.

```suggestion
        if (empty($paymentAction[$data])) {
            return '';
        }
        
        return $paymentAction[$data];
```